cluster/gce/gci/configure-helper.sh

```diff
@@ -2164,7 +2164,13 @@ function start-cluster-autoscaler {
 
     local params="${AUTOSCALER_MIG_CONFIG} ${CLOUD_CONFIG_OPT} ${AUTOSCALER_EXPANDER_CONFIG:---expander=price}"
     params+=" --kubeconfig=/etc/srv/kubernetes/cluster-autoscaler/kubeconfig"
-    sed -i -e "s@{{params}}@${params}@g" "${src_file}"
+
+    # split the params into separate arguments passed to binary
+    local params_split
+    params_split=$(eval "for param in $params; do echo -n \\\"\$param\\\",; done")
```

Comment:
Why do you need the eval? Seems like `params_split=$(for param in $params; do echo -n \"$param\",; done)` would give you the same result, no?

Reply:
Turns out it doesn't do the same if the params themselves contain quotes, which I didn't think about.